src/routes/test/[slug]/page.test.ts

```
@@ -0,0 +1,200 @@
+			testQuestions: null,
+			timeToBegin: 10,
+			testData: mockTestData
+		});
+	});
+});
+
+describe('Action', () => {
+	describe('Create Candidate', () => {
+		const mockFetch = vi.fn();
+		const mockCookies = { set: vi.fn() };
+		const mockRequest = {
```

ESLint

[error] 'mockRequest' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)